example/ZXingQtReader.h

```diff
@@ -408,6 +406,20 @@ public slots:
 	void failedRead();
 	void foundBarcode(ZXingQt::Barcode barcode);
 
+	void formatsChanged();
```

Comment:
Is this implicitly a `Q_SIGNAL` when mentioned in a `Q_PROPERTY` call?

Reply:
This is a signal because it is in a `signals` section (starts at line 405), not because it is mentioned in `Q_PROPERTY`.

You can mark a function as a signal by either
 - placing it in a signals sections (starts with `signals:` or `Q_SIGNALS:` (not the S at the end)
 - writing `Q_SIGNAL` in front of the member declaration.

Qt itself mostly uses the first option.